design/kata-2-0-metrics.md

```diff
@@ -0,0 +1,307 @@
+| `kata_magent_go_memstats_heap_sys_bytes`: <br> Number of heap bytes obtained from system. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_last_gc_time_seconds`: <br> Number of seconds since 1970 of last garbage collection. | `GAUGE` | seconds |  | 2.0.0 |
+| `kata_magent_go_memstats_lookups_total`: <br> Total number of pointer lookups. | `COUNTER` |  |  | 2.0.0 |
+| `kata_magent_go_memstats_mallocs_total`: <br> Total number of `mallocs`. | `COUNTER` |  |  | 2.0.0 |
+| `kata_magent_go_memstats_mcache_inuse_bytes`: <br> Number of bytes in use by `mcache` structures. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_mcache_sys_bytes`: <br> Number of bytes used for `mcache` structures obtained from system. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_mspan_inuse_bytes`: <br> Number of bytes in use by `mspan` structures. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_mspan_sys_bytes`: <br> Number of bytes used for `mspan` structures obtained from system. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_next_gc_bytes`: <br> Number of heap bytes when next garbage collection will take place. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_other_sys_bytes`: <br> Number of bytes used for other system allocations. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_stack_inuse_bytes`: <br> Number of bytes in use by the stack allocator. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_stack_sys_bytes`: <br> Number of bytes obtained from system for stack allocator. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_memstats_sys_bytes`: <br> Number of bytes obtained from system. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_go_threads`: <br> Number of OS threads created. | `GAUGE` |  |  | 2.0.0 |
+| `kata_magent_process_cpu_seconds_total`: <br> Total user and system CPU time spent in seconds. | `COUNTER` | seconds |  | 2.0.0 |
+| `kata_magent_process_max_fds`: <br> Maximum number of open file descriptors. | `GAUGE` |  |  | 2.0.0 |
+| `kata_magent_process_open_fds`: <br> Number of open file descriptors. | `GAUGE` |  |  | 2.0.0 |
+| `kata_magent_process_resident_memory_bytes`: <br> Resident memory size in bytes. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_process_start_time_seconds`: <br> Start time of the process since Unix epoch in seconds. | `GAUGE` | seconds |  | 2.0.0 |
+| `kata_magent_process_virtual_memory_bytes`: <br> Virtual memory size in bytes. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_process_virtual_memory_max_bytes`: <br> Maximum amount of virtual memory available in bytes. | `GAUGE` | bytes |  | 2.0.0 |
+| `kata_magent_running_shim_count`: <br> Running shim count(running sandboxes). | `GAUGE` |  |  | 2.0.0 |
+| `kata_magent_scrape_count`: <br> Scape count. | `COUNTER` |  |  | 2.0.0 |
+| `kata_magent_scrape_durations_histogram_million_seconds`: <br> Time used to scrape from shims | `HISTOGRAM` | million seconds |  | 2.0.0 |
```

Comment:
Is this really "million" (`1,000,000`) or should it be "milli" (`1/1,000`) (here and below):

Reply:
Yes, it's should be `millisecond` 👍